_config.yml

```diff
@@ -9,7 +9,8 @@ plugins:
 relative_links:
   enabled:     true
   collections: false
-last_edit_timestamp: true # show or hide edit time - page must have `last_modified_date` defined in the frontmatter
-last_edit_time_format: "%e %B %Y" # uses ruby's time format: https://ruby-doc.org/stdlib-2.7.0/libdoc/time/rdoc/Time.html
```

Comment:
@venusbb thanks for adding the edit links on those pages. Hopefully, that will make it slightly easier for community members or suppliers to contribute :)

Removing these lines of code from the config file remove the last updated date from the footer of pages that have defined it in the frontmatter. This change, therefore, doesn't affect most pages but affects the two pages I published about the GDS discovery. For these pages, I think the date is quite useful. Although there are probably multiple ways to add a date, I guess this might be a way to do it consistently across pages? I'm open to alternatives if you think we should use a different approach?